src/test/1_OracleHub.t.sol

```diff
@@ -842,7 +842,7 @@ contract OracleHubTest is Test {
         assertEq(expectedRateInBaseCurrency, actualRateInBaseCurrency);
     }
 
-    function testReturnUsdRateWhenBaseCurrencyIsNotUsdOverflow1(
+    function testRevert_getRate_WhenBaseCurrencyIsNotUsdOverflow1(
```

Comment:
testRevert_getRate_BaseCurrencyNotUsd_Overflow
is better naming
in general filler words like 'should' 'when' are not that necessary in test naming.
general convention for test naming is [unit_of_work][scenario][expected_result], in our case we should always start with test... to the test naming. It is good to use testRevert or testSuccess. but also good to add expected_result at the end of the test name.